test/powershell/engine/Basic/Assembly.LoadWithPartialName.Tests.ps1

```diff
@@ -6,6 +6,8 @@ Describe "Assembly::LoadWithPartialName Validation Test" -Tags "CI" {
     $defaultErrorId = 'FileLoadException'
     $testcases = @(
         # verify winforms is blocked
+        # winforms assembly is supported for .Net Core 3.0, if a new assembly needs to be blocked
+        # enable this test and add to list below
         @{
             Name = 'system.windows.forms'
```

Comment:
Should we remove this?

Reply:
I wanted to keep this here in case there's assemblies in the future that should be skipped